packages/mobile/src/paymentRequest/saga.ts

```diff
@@ -0,0 +1,191 @@
+import firebase from '@react-native-firebase/app'
```

Comment:
No new code here, all moved from src/firebase/saga.ts
Only very minor edits and one new function at the bottom (see next comment)